cmstestsuite/TestCleanCheckout.py

```diff
@@ -77,7 +77,8 @@ def setup_cms():
     os.environ["PYTHONPATH"] = "%(TEST_DIR)s" % CONFIG
 
     info("Building cms.")
-    sh("./setup.py build")
+    sh("./prerequisites.py compile_l10n")
+    sh("./prerequisites.py compile_isolate")
```

Comment:
If desired, one could even add a custom build command (so that the pollution is mitigated), like [this](https://gist.github.com/wil93/e8a4aad9205ab9e772c0). However, I still feel that it's better to separate it (e.g. in this way you would need to run 3 commands in sequence, instead of 2).